web/src/message_util.js

```diff
@@ -20,6 +20,11 @@ export function add_messages(messages, msg_list, opts) {
     if (!messages) {
         return undefined;
     }
+    // Don't proceed with rendering logic on an undefined
+    // table name (i.e., the recent topics list)
+    if (msg_list.table_name === undefined) {
+        return undefined;
+    }
```

Comment:
The call we’re skipping here is `msg_list.add_messages(messages, opts)`, which is not just for the DOM; it also updates `MessageListData`.

I think `recent_topics_message_list` should just not have a `MessageListView`.

Reply:
Yeah I think the intent had been that `recent_topics_message_list` would just be a `MessageListData` without the other classes.